app/models/feed.rb

```diff
@@ -10,7 +10,13 @@ def get(limit, max_id = nil, since_id = nil)
     if redis.exists("account:#{@account.id}:regeneration")
       from_database(limit, max_id, since_id)
     else
-      from_redis(limit, max_id, since_id)
+      statuses = from_redis(limit, max_id, since_id)
+
+      statuses += from_database(limit - statuses.size,
```

Comment:
This query will be executed even if `limit-statuses.size` is 0. I think this is a useless query execution.